packages/api/package.json

```diff
@@ -66,7 +66,7 @@
     "@types/compression": "^1.7.5",
     "@types/express": "^4.17.21",
     "@types/lodash": "^4.14.202",
-    "@types/node": "^20.10.5",
+    "@types/node": "^20.14.9",
```

Comment:
Do we need to change this node version???

Reply:
Need to? No, it was one of the things I tried to improve things. I did not think it was significant. I will revert.